hackathon_site/dashboard/frontend/src/components/teamDetail/TeamInfoTable/TeamInfoTable.tsx

```diff
@@ -53,7 +51,15 @@ export const TeamInfoTable = () => {
                                         {`${row.user.first_name} ${row.user.last_name}`}
                                     </TableCell>
                                     <TableCell>{row.user.email}</TableCell>
-                                    <TableCell>802-207-9999</TableCell>
+                                    <TableCell>
+                                        {`${row.phone_number.substring(
```

Comment:
This is fine but some people add their area code in front of their phone number (we allow it in their application) so we have to handle cases for that.

Maybe try implementing something like this: https://learnersbucket.com/examples/javascript/how-to-format-phone-number-in-javascript/ 
but incorporate the regex from our application form

Reply:
<img width="203" alt="CleanShot 2022-09-15 at 01 11 26@2x" src="https://user-images.githubusercontent.com/36619969/190319790-3e30d8ae-5095-4908-a937-84d70f71d122.png">

Reply:
ok so i added a little phone number formatter thing. you can play with it here https://jsfiddle.net/cst43qku/12/

it's quite simple, but should work for the phone numbers in the application. basically i assume the phone number is valid (it's checked against the regex validator in the registration model) and just get all the digits. i then map the last 10 to (xxx) xxx-xxxx, and anything before that i put a + before and call it the international code.

it won't work for numbers that don't follow the (xxx) xxx-xxxx pattern, but i don't think our registration regex even allows these, and if it does then this problem gets really annoyingly hard without using an external library.

i hate regex, lmk what u think

Reply:
honestly, great job on this 👏👏👏